src/hooks/useRejectionToast.tsx

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from "react";
+
+import { useToasts } from "../design-system/Toast";
+
+const useRejectionToast = () => {
+  const { addToast } = useToasts();
+
+  return useCallback(
+    async (p: Promise<any>) => {
+      try {
+        return await p;
+      } catch (e) {
+        addToast(e.message);
```

Comment:
Since this is an error toast, we probably want to change the color from green to red? You can pass an "appearance" param here and Toast will take care of that.

Reply:
oh good note, totally missed that!